lib/OpenQA/Utils.pm

```diff
@@ -762,14 +695,13 @@ sub loaded_modules {
 
 # Fallback to loaded_modules if no arguments are given.
 # Accepts namespaces as arguments. If supplied, it will filter by them
-sub loaded_plugins {
-    my $ns = join('|', map { quotemeta } @_);
-    return @_ ? grep { /^$ns/ } loaded_modules() : loaded_modules();
+sub loaded_plugins (@ns) {
```

Comment:
Is `@ns` a namespace? I think is something more unrpocessed, should it deserve better name?

Reply:
Again, not related to introducing signatures. The variable was called `@ns` and is still called `@ns` in the signature